packages/backend-core/src/objectStore/objectStore.ts

```diff
@@ -105,37 +110,43 @@ export function ObjectStore(
  * Given an object store and a bucket name this will make sure the bucket exists,
  * if it does not exist then it will create it.
  */
-export async function makeSureBucketExists(client: any, bucketName: string) {
+export async function createBucketIfNotExists(
+  client: any,
+  bucketName: string
+): Promise<boolean> {
   bucketName = sanitizeBucket(bucketName)
   try {
     await client
       .headBucket({
         Bucket: bucketName,
       })
       .promise()
+    return false
```

Comment:
This return might be confusing? It means it exists, that it was created...? If needed, can we type it to something like `return { created: false, exists: true}`?

Reply:
I'd originally suggested to Peter to rename this function to make the return a bit more clear, it says whether or not the file was created. You're right, though, it could just as easily represent if it already existed. Not sure how to make this function hard to use incorrectly.